formats/opentype.fathom

```diff
@@ -198,10 +207,77 @@ let language_id =
 let cmap_language_id = fun (platform : Repr platform_id) =>
     language_id;
 
+// cmap sub-table format 8 has a 32-bit language code
+let cmap_language_id32 = fun (platform : Repr platform_id) =>
+    language_id32;
```

Comment:
Why is this parameterised by platform?

Reply:
Ahh I meant to comment on that. There might be a better way to do this.

It's parameterised by platform because I just copied the 16-bit version that was already there. https://github.com/yeslogic/fathom/pull/313/files#diff-fd2bf33c4274b68a3b5148dafc986eafbff4ac002bb464422b4178c92d420834R207

Reply:
I think Brendan parameterised it by platform because the interpretation of the language depends on the platform but I don't think that influences parsing.

Reply:
Yeah, it's more about how this information is interpreted (I forgot to add a TODO). See the [MS OpenType Spec](https://docs.microsoft.com/en-us/typography/opentype/spec/cmap#use-of-the-language-field-in-cmap-subtables) for more information. We could always remove it for now though.